dashboard/test/ui/features/videoplayer_eyes.feature

```diff
@@ -32,7 +32,7 @@ Scenario: Fallback player for embedded
   And I close my eyes
 
 # Starting in Chrome 62, sites can no longer automatically run plugins.
-@chrome_before_62
+@no_chrome
```

Comment:
I confirmed in https://drone.cdn-code.org/code-dot-org/code-dot-org/843/2/5 that this feature still does not work in latest chrome.